client/components/push-notification/prompt.jsx

```diff
@@ -79,13 +80,25 @@ const PushNotificationPrompt = React.createClass( {
 		return <Notice className="push-notification__notice" text={ noticeText } icon="bell" onDismissClick={ this.props.dismissNotice } />;
 	},
 
+	isUnsupportedChromeVersion: function() {
+		if ( global.window && global.window.chrome && global.window.navigator.appVersion ) {
+			const chromeVersion = global.window.navigator.appVersion.match( /Chrome\/(\d+)/ )[ 1 ];
+			if ( chromeVersion < 50 ) {
+				analytics.mc.bumpStat( 'calypso_push_notification_chrome', chromeVersion );
```

Comment:
Good idea to track this with a stat!

The stat name seems a bit vague (people looking at the stat wouldn't know that it only tracks unsupported versions of Chrome). I'd suggest renaming it to something like `calypso_push_notification_unsupported_chrome` (not sure if that is too long -- you'll want to double-check that).